dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -44,29 +44,6 @@
     = code_org_link + "'s"
     CS Fundamentals workshop.
 
-  - if @dash_code
```

Comment:
Can we remove setting `@dash_code = CDO.pd_workshop_exit_survey_dash_code` from `dashboard/app/mailers/pd/workshop_mailer.rb` as part of this change? Assuming so, can we clean up the `CDO.pd_workshop_exit_survey_dash_code` secret?